docs/quickstarts/chrome-extension.mdx

```diff
@@ -209,114 +206,89 @@ description: Add authentication and user management to your Chrome Extension wit
     )
   }
 
-export default IndexPopup
+  export default IndexPopup
   ```
 
-  ## Create a consistent CRX ID for your extension
+  ### Create a consistent CRX ID for your extension
 
-  Chrome Extensions have a unique CRX ID that rotates by default, which can cause errors and issues with the Clerk integration. To avoid these problems, ensure that you have a **consistent** CRX ID in both development and production for your extension by following these steps:
+  Chrome Extensions have a unique CRX ID that rotates by default, which can cause errors with the Clerk integration. To avoid these problems, ensure that you have a **consistent** CRX ID in both development and production for your extension by following these steps:
 
   1. Visit Plasmo Itero's [Generate Keypairs](https://itero.plasmo.com/tools/generate-keypairs) tool.
-  1. Select the **Generate KeyPairs** button.
-  1. Save the private key somewhere safe in case you need it in the future. Save the public key for use in the next step, and the CRX ID for use in a later step.
+  1. Select **Generate KeyPairs**.
+  1. Save the **Private Key** somewhere secure in case you need it in the future. Save the **Public Key** and the **CRX ID** for the next steps.
 
   ### Create an `.env.chrome` file to store your public key
 
-  Create an `.env.chrome` file and add your public key to it.
+  Create an `.env.chrome` file and add your public key to it, as shown in the following example:
 
   ```env {{ filename: '.env.chrome' }}
-  CRX_PUBLIC_KEY="<PUBLIC KEY>"
+  CRX_PUBLIC_KEY=<YOUR_PUBLIC_KEY>
   ```
 
   ### Edit your `package.json` to use the new public key
 
-  If you were building an extension from scratch or using other tools, you would need to maintain a `manifest.json` file, and potentially modify it for development and production. Plasmo uses the `package.json` to generate a `manifest.json` on build, and allows for the use of environment variables in `package.json`.
+  Plasmo uses the `package.json` to generate a `manifest.json` on build, and allows for the use of environment variables in `package.json`.
```

Comment:
Thoughts on linking to https://docs.plasmo.com/framework#where-is-the-manifestjson-file here, as the info about this was removed?

Reply:
```suggestion
  Plasmo [uses the `package.json` to generate a `manifest.json` on build](https://docs.plasmo.com/framework#where-is-the-manifestjson-file), and allows for the use of environment variables in `package.json`.
```